app/org/sagebionetworks/bridge/play/controllers/ConsentController.java

```diff
@@ -39,39 +39,53 @@ public Result getConsentSignature() throws Exception {
         return okResult(sig);
     }
 
-    @Deprecated
-    public Result giveV1() throws Exception {
-        return giveConsentForVersion(1);
-    }
-
-    public Result giveV2() throws Exception {
+    public Result giveV3() throws Exception {
         return giveConsentForVersion(2);
```

Comment:
Consistency note: should be giveConsentForVersion(3). Following the call chain, it only special cases version 1, so there should be no other code change required.